js/ControlTodoItem.js

```diff
@@ -0,0 +1,28 @@
+const $todoList = document.getElementById("todo-list");
+
+export function initControlTodoItem(){
+    $todoList.addEventListener("click", onToggleTodoItem);
+}
+
+//완료한 일로 check, 할 일 삭제 : edit모드에서는 작동X
+function onToggleTodoItem(e) {
+    const li = e.target.closest("li");
+
+    //할 일을 체크하거나 해제
+    if(li.classList.value === "" || li.classList.value === "completed"){
+        //완료된 일 
+        if(e.target && e.target.nodeName === 'INPUT'){ 
+            e.target.closest("li").classList.toggle("completed");
+        }
+        
+        //삭제
+        if(e.target && e.target.nodeName === 'BUTTON'){
+            var returnValue = confirm('정말로 삭제하시겠습니까?');
+            if(returnValue === true){
```

Comment:
```suggestion
            if(returnValue){
```

이렇게만 써도 true일 경우 if문을 실행하고 false일 경우 벗어난답니다 😆

Reply:
위의 피드백과 함께 수정하여 review#13에 반영하였습니다. 👂 
좋은 피드백 감사합니다. 😃